src/Shared/AzureManaged/Shared.AzureManaged.csproj

```
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net6.0</TargetFrameworks>
```

`TargetFrameworks` (with an 's') is for multi-targeting.

```suggestion
    <TargetFramework>net6.0</TargetFramework>
```